src/main/java/io/xpring/xrpl/DefaultXRPClient.java

```diff
@@ -292,11 +292,44 @@ public boolean accountExists(String address) throws XRPException {
     }
   }
 
-  @Override
-  public int getLatestValidatedLedgerSequence() throws XRPException {
+  public int getOpenLedgerSequence() throws XRPException {
     return this.getFeeResponse().getLedgerCurrentIndex();
   }
 
+  /**
+   * Retrieve the latest validated ledger sequence on the XRP Ledger.
+   * <p>
+   * Note: This call will throw if the given account does not exist on the ledger at the current time. It is the
+   * *caller's responsibility* to ensure this invariant is met.
+   * </p><p>
+   * Note: The input address *must* be in a classic address form. Inputs are not checked to this internal method.
+   * </p><p>
+   * TODO(keefertaylor): The above requirements are onerous, difficult to reason about and the logic of this method is
+   * brittle. Replace this method's implementation when rippled supports a `ledger` RPC via gRPC.
+   * </p>
+   * @param address An address that exists at the current time. The address is unchecked and must be a classic address.
+   * @return The index of the latest validated ledger.
+   * @throws XRPException If there was a problem communicating with the XRP Ledger.
+   */
+  @Override
+  public int getLatestValidatedLedgerSequence(String address) throws XRPException {
+    // rippled doesn't support a gRPC call that tells us the latest validated ledger sequence. To get around this,
+    // query the account info for an account which will exist, using a shortcut for the latest validated ledger. The
+    // response will contain the ledger the information was retrieved at.
```

Comment:
I would say "... contain the ledger **index** the information ... " (or "sequence" ... )